manageiq-api-client.gemspec

```diff
@@ -24,8 +24,14 @@ Gem::Specification.new do |spec|
   spec.require_paths = ["lib"]
 
   spec.add_dependency "activesupport", ">= 6.0", "<7.1"
-  spec.add_dependency "faraday", "~> 2.9"
-  spec.add_dependency "faraday-follow_redirects"
```

Comment:
faraday-follow-redirects is written in a way that supports both v1 and v2, so we can just add the dependency directly.